control/state.py

```diff
@@ -486,8 +486,8 @@ def _watcher_callback(notify_id, notifier_id, watch_id, data):
         if self.watch is None:
             try:
                 self.watch = self.ioctx.watch(self.omap_name, _watcher_callback)
-            except Exception as ex:
-                self.logger.error(f"Unable to initiate watch: {ex}")
+            except Exception:
+                self.logger.exception(f"Unable to initiate watch")
```

Comment:
Maybe raise here?

Reply:
@baum , notice that even with a failure here we will still run update() using the timer. Anyway, I opened #594 for this issue.